cidemipyme-gatsby/package.json

```diff
@@ -10,7 +10,7 @@
     "gatsby-plugin-react-helmet": "^2.0.5",
     "gatsby-source-filesystem": "^1.5.25",
     "gatsby-transformer-json": "^1.0.16",
-    "jquery": "^3.0.0",
+    "jquery": "^3.3.1",
```

Comment:
I think in this case you could consider using axios, it is like 7x smaller than jquery. Also, I think you are not using web pack externals, with this query will not be bundled, instead it will be loaded from the cdn.